etc/config/go.amazon.properties

```diff
@@ -52,6 +52,24 @@ compiler.gl1110.exe=/opt/compiler-explorer/golang-1.11/go/bin/go
 compiler.gl1110.semver=1.11
 compiler.gl1110.goroot=/opt/compiler-explorer/golang-1.11/go
 
+###############################
+# Cross GO
+group.cross.compilers=&ppc
+group.cross.supportsBinary=false
+group.cross.groupName=Cross Fortran
+
+###############################
+# GCC for PPC
+group.ppc.compilers=ppc64leg8:ppc64g8
+group.ppc.groupName=POWER Compilers
+group.ppc.isSemVer=true
+compiler.ppc64leg8.exe=/opt/compiler-explorer/powerpc64le/gcc-at12/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-gccgo
+compiler.ppc64leg8.name=power64le Advance Toolchain 12.0 (gcc branch ibm/gcc-8-branch)
```

Comment:
I think the names for this compilers might be a bit too long, but nothing too worriesome. If you could come up with a shorter version (Are all the naming details needed to properly differenciate them? Or could some of that info be just, if needed, gathered from 
![imagen](https://user-images.githubusercontent.com/5364255/48431779-d7f13f80-e772-11e8-9ae4-d328b583a74c.png) which shows the version as reported by the compiler version flag?)

Reply:
Thanks for review. Name shorted.. Quite right, the version flag is:
<pre>
$ powerpc64le-unknown-linux-gnu-gccgo --version
powerpc64le-unknown-linux-gnu-gccgo (crosstool-NG 1.23.0.471-ad86) 8.2.1 20180727 Advance Toolchain AT12, based on subversion id 263037
</pre>